tests/integration/calculator_windows_py2.txt

```diff
@@ -0,0 +1,17 @@
+nose.config: INFO: Ignoring files matching ['^\\.', '^_', '^setup\\.py$']

```

Comment:
I suppose this file should really be the expected output on Python 3 as well... not sure why `check_output()` doesn't currently return it though.